src/app/api/routes/installations.py

```diff
@@ -31,3 +32,13 @@ async def update_installation(payload: InstallationIn, installation_id: int = Pa
 @router.delete("/{installation_id}/", response_model=InstallationOut)
 async def delete_installation(installation_id: int = Path(..., gt=0)):
     return await crud.delete_entry(installations, installation_id)
+
+
+@router.get("/list_devices/{site_id}", response_model=List[int])
```

Comment:
For consistency, I'd recommend changing "list_devices" to "list-devices", and moving the site_id before "list-devices" 

And are we positive that using typing as response model is supported? (not schemas I mean) Have you tried the feature locally?

Reply:
I  support @frgfm point of view. I would create a dedicated schema to  make sure. In addition, it will make it clearer what the route returns.

Reply:
Now that I think of it, if we are in the installations routes, it would be weird to have `/installations/{site_id)]/liste-devices` knowing that `/installations/{id}` is to retrieve an installation.
That could be confusing. maybe `/installations/site-devices/{site_id}`

Reply:
I agree with your last suggestion!

Reply:
OK